conf/test_data.config

```diff
@@ -1,88 +1,809 @@
+// README:
+// https://github.com/nf-core/test-datasets/blob/modules/README.md
+
 params {
-    // Base directory for test data
-    test_data_base = "https://raw.githubusercontent.com/nf-core/test-datasets/phageannotator"
-    modules_testdata_base_path = "s3://ngi-igenomes/testdata/nf-core/modules/"
+    // Base directories for test data
+    phageannotator_test_data_base = "https://raw.githubusercontent.com/nf-core/test-datasets/phageannotator"
+    test_data_base = 'https://raw.githubusercontent.com/nf-core/test-datasets/modules'
```

Comment:
Looking through the rest of the file, is there a reason why you are copying over what looks like the entire test data config? 

I wonder if you can just import the file with the map itself rather than making a copy

Reply:
I did just copy over the file because I wasn't aware of better options. I'm definitely open to alternatives! 

What would importing the file with the map itself look like?